tests/packages/sharedlib-in-package/mypkg/__init__.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: 2024 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import sys
+
+
+# Note: the below function is used in the docs via a literalinclude directive.
+#       When making any changes to this file, please ensure the right set of
+#       lines is pulled into the docs, and adjust the :lines: attribute of
+#       the literalinclude if needed.
```

Comment:
I think that Sphynx can include Python code by object name, so the issue describe here can be avoided using
```rest
.. literlainclude:: .../__init__.py
   pyobejct: _enable_sharedlib_loading
```
or something like this, maybe?

Reply:
The `start-after` and `end-before` arguments for `literalinclude` are also an option that does not require to hard-code line numbers https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-option-literalinclude-start-after

Reply:
Thanks, fixed now - much easier to maintain this way.